packages/cel/manifest.yml

```
@@ -90,6 +90,27 @@ policy_templates:
         default: |
           #products: '(?i)(Elasticsearch|Beats|Logstash|Kibana)'
           #solutions: '(?i)(Search|Observability|Security)'
+      - name: xsd
+        type: textarea
+        title: XML Schema Definitions (XSD) for XML documents.
+        description: |
+          The XSD needed for correct parsing and ingestion of XML documents using decode_xml CEL extension.
+          More information can be found in the [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-cel.html#xsd-cel).
+        show_user: true
```

```suggestion
        show_user: false
```
This in not a common enough use case for us to push it on everyone.

---

updated as suggsted.